### PR TITLE
issue-110: Keep aspect ratio of images, and clip them based on phone/…

### PR DESCRIPTION
**Pull request by jperedadnr**
…tablet factor

The images on a tablet are clipped from the center.

This is how it looks like on:

Android phone

Android phone

Android tablet
Android tablet

iOS phone
iOS phone

iOS tablet
iOS tablet

### CODE REVIEW

File: DevoxxClientMobile/src/main/java/com/devoxx/views/cell/ConferenceCell.java [hunk 1]
```diff
@@ -135,15 +143,14 @@ public void updateItem(Conference item, boolean empty) {
             imageTask = new ETagImageTask("conference_" + item.getId(), item.getImageURL());
             imageTask.setOnSucceeded(e -> {
                 background.setImage(imageTask.getValue());
+                fitImage();
             });
             imageTask.exceptionProperty().addListener((o, ov, nv) -> {
                 LOG.log(Level.SEVERE, nv.getMessage());
             });
             executor.submit(imageTask);
             imageTask.image().ifPresent(background::setImage);
```
Comment (tiainen) [inline DevoxxClientMobile/src/main/java/com/devoxx/views/cell/ConferenceCell.java]: Does `fitImage()` need to be called each time the background image is set?

Comment (jperedadnr) [reply DevoxxClientMobile/src/main/java/com/devoxx/views/cell/ConferenceCell.java]: Yes, as each image has different dimensions

Comment (tiainen) [reply DevoxxClientMobile/src/main/java/com/devoxx/views/cell/ConferenceCell.java]: What I meant is that a call to fitImage is missing then at this line, inside the ifPresent lambda.

Comment (jperedadnr) [reply DevoxxClientMobile/src/main/java/com/devoxx/views/cell/ConferenceCell.java]: From what I've tested, the image is always set on succeed of the task (`imageTask.image()` is always empty).

**Comment by jperedadnr** [comment]
@tiainen I've added the change you required, noticing that there is a TODO pending on ETagImageTask.

**Pull request merged**